gsap.d.ts

```diff
@@ -465,7 +465,7 @@ declare namespace GSAPStatic {
 	   * @returns {Timeline}
 	   * @memberof Timeline
 	   */
-		add(value: Tween | Timeline | string | string[], position?: positionType, align?: string, stagger?: number | string): this;
+		add(value: Tween | Tween[] | Timeline | Timeline[] | string | string[], position?: positionType): this;
```

Comment:
The new add() also takes a callback. The orginal is a mistake of mine, not having it.

```suggestion
		add(value: Tween | Tween[] | Timeline | Timeline[] | string | string[] | callbackFn, position?: positionType): this;
```

Reply:
Ok I will fix it in the next batch 😄